### PR TITLE
✅ use go module version paths

### PR DESCRIPTION
**Pull request by faddat**
This PR uses standard go module versioning so that others don't accidentally use the wrong version of juno when building stuff on or related to juno.

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -46,6 +46,6 @@ jobs:
         with:
           context: .
           platforms: linux/amd64
-          push: true
+          push: ${{ github.event_name != 'pull_request' }}
```
Comment (the-frey) [inline .github/workflows/docker.yml:49]: Please remove this as it isn't relevant to this PR 👍

Comment (the-frey) [reply .github/workflows/docker.yml:49]: Other than that, looks good.

**Comment by faddat** [comment]
Force push context:

instead of updating <stuff> I just checked out main and applied the updates directly to it.  This should be ready to rock.

**Review by the-frey — APPROVED** [review]
LGTM

**Review by joeabbey — APPROVED** [review]
vNice

**Pull request merged**